test/integration/build/aws.tf

```diff
@@ -1558,22 +1560,23 @@ resource "aws_rds_cluster" "rds_cluster" {
 }
 
 resource "aws_rds_cluster_instance" "instance1" {
-  count               = var.aws_enable_creation
+  count              = var.aws_enable_creation
   apply_immediately  = true
   cluster_identifier = aws_rds_cluster.rds_cluster.0.cluster_identifier
   identifier         = "instance1"
```

Comment:
wonder if we should create a unique name/identifier for these clusters to avoid clashes if someone else has this setup.

Reply:
I think every resource should have a unique identifier showing from which test plan they are created from, e.g. `aws-inspec-resource-name-some-random-string`

Reply:
Actually, for the rds resource, terraform creates a unique identifier if not provided. Maybe we can just delete the identifier in this resource.
https://www.terraform.io/docs/providers/aws/r/rds_cluster_instance.html
```
identifier - (Optional, Forces new resource) The identifier for the RDS instance, if omitted, Terraform will assign a random, unique identifier.
```

Reply:
Seems reasonable

Reply:
Added unique identifiers due to being used in integration tests.